exporter/awss3exporter/s3_writer.go

```diff
@@ -60,6 +64,18 @@ func newUploadManager(
 		})
 	}
 
+	if retryMaxAttempts := conf.S3Uploader.RetryMaxAttempts; retryMaxAttempts != 0 {
```

Comment:
https://docs.aws.amazon.com/sdk-for-go/v2/developer-guide/configure-retries-timeouts.html says that zero is meaningful:
`Setting max attempts to zero will allow the SDK to retry all retryable errors until the request succeeds, or a non-retryable error is returned.`